server/src/main/java/org/apache/druid/server/metrics/QueryCountStatsMonitor.java

```diff
@@ -49,11 +49,15 @@ public boolean doMonitor(ServiceEmitter emitter)
     final long interruptedQueryCount = statsProvider.getInterruptedQueryCount();
     Map<String, Long> diff = keyedDiff.to(
         "queryCountStats",
-        ImmutableMap.of("query/count", successfulQueryCount + failedQueryCount + interruptedQueryCount,
-                        "query/success/count", successfulQueryCount,
-                        "query/failed/count", failedQueryCount,
-                        "query/interrupted/count", interruptedQueryCount
-
+        ImmutableMap.of(
+            "query/count",
```

Comment:
You could keep corresponding keys and values on the same line, e. g.
```java
ImmutableMap.of(
    key1, value1,
    key2, value2,
    ...
)
```
I think it's more readable and recognised as an exception from the general rule here: https://github.com/apache/incubator-druid/pull/6461/files#diff-b26be0568e2252c7501570f97d7a91e8R212

Reply:
Yes

Reply:
Updated, thanks.